### DISCUSSION TITLE
block inserter how can i keep it open?

### DISCUSSION THREAD

**Discussion by sinanisler** [WordPress/gutenberg#46338] [Q&A]
how can i keep block inserter open what js object ? or code?

read lots of developer site cant find anything about it. there is only block examples. 
how can i control the block inserter itself ?

i just want to keep it open all the time.

thanks

**Comment by aburchette** [comment]
Please provide a solution to thing. Very frustrating that it closes after a selection.

**Comment by stokesman** [comment]
Here’s a simple hack (in plugin form) that works in my testing:
https://gist.github.com/stokesman/af6cf3a3f9d068e21832084141a81c05
